src/lib/CellTemplates/ChevronCellTemplate.tsx

```diff
@@ -100,7 +100,7 @@ export class ChevronCellTemplate implements CellTemplate<ChevronCell> {
                     }}
                     defaultValue={cell.text}
                     onChange={e => onCellChanged(this.getCompatibleCell({ ...cell, text: e.currentTarget.value }), false)}
-                    onBlur={e => onCellChanged(this.getCompatibleCell({ ...cell, text: e.currentTarget.value }), true)}
+                    onBlur={e => onCellChanged(this.getCompatibleCell({ ...cell, text: e.currentTarget.value }), (e as any).view?.event?.keyCode === keyCodes.ESCAPE ? false : true)}
```

Comment:
No need for the ternary expression, just go `(e as any).view?.event?.keyCode !== keyCodes.ESCAPE`.
Also, is there maybe a way to ditch the `any`? If not, or not that easy, just leave it.

Reply:
Also applies to the occurrences below :)

Reply:
I removed the fourth occurrences of the ternary expression.
Regarding to the 'any' type, I tried to avoid it yesterday with no success. Sorry.

and thanks for the review!